raster-foundry/raster-foundry
# Rename tool -> template, tool-run -> analysis

## Overview
Call Tools Templates and ToolRuns Analyses
Clean up inconsistencies in copy

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

Moved some files around so the naming conventions make more sense, and consolidated a few component/controller/module collections

## Testing Instructions
Verify that functionality is unchanged:
Template creation
Template browsing
Create Analysis from template
Edit / Update / View an Analysis

Closes https://github.com/raster-foundry/raster-foundry/issues/2760
Closes https://github.com/raster-foundry/raster-foundry/issues/2759
